CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+Allocate gridded fields to use the pinflag option needed for the Single System Image  (SSI) capability.
```

Comment:
Can you consolidate all the `Changed` bits to entries under the "main" `### Changed` entry after `### Added`?

Or I can do it for you if you like.

Reply:
Never mind. I had to do it to update branch anyway.